### ISSUE TITLE
Please expose user information for User pages

### ISSUE THREAD

**Issue by filbranden** [jekyll/github-metadata#149]
I'm trying to access information about the GitHub user from a User gh-pages template, information such as the full name, location, company, etc.

More specifically, I would like to see the fields in [this response](https://developer.github.com/v3/users/#response).

I tried looking at `{% for user in site.github.organization_members %}`, expecting that would be the one user that owns the User pages, but that's empty... Then I tried `{% for user in site.github.contributors %}`, but that only lists the base information (such as the username), which might be OK if I had access to making more API requests, but from gh-pages I really can't...

Can we expose a field that would be only valid on User pages and that would be populated with that response for the owner of that site?

**NOTE:** I'm happy to try to contribute, though Ruby isn't really my thing and I'm unsure of how to actually test this... If you can give me some pointers, I can try to hack something together to make this work.

Thanks!
Filipe

**Comment by parkr** [comment]
> Can we expose a field that would be only valid on User pages and that would be populated with that response for the owner of that site?

I could see adding `site.github.owner`, which exposes some user data about the owner of the repository. If it's an org, then we'd get org information. We'd likely want to limit this somewhat to prevent abuse, but public information like full name, location, and company should all be safe to include.

**Comment by filbranden** [comment]
It seems today ["setting a status"](https://help.github.com/articles/personalizing-your-profile/#setting-a-status) is a thing...

I think I'd like to expose that too, for user pages (?). I looked at the User object and didn't see that information there (yet, but maybe I'm looking at the wrong place.)

Anyways, we can discuss that after #151 has been reviewed and merged...

Cheers!
Filipe

**Comment by parkr** [comment]
Let's hold off on exposing statuses for now. Let's open a separate issue on that to track it and get input.
